src/cli/README_COMMISSIONER.md

```diff
@@ -120,6 +121,22 @@ Done
 Done
 ```
 
+### joiner table
+
+Usage: `commissioner joiner table`
+
+List all Joiner entries.
+
+```bash
+> commissioner joiner table
+| ID                    | PSKd                             | Expiration |
++-----------------------+----------------------------------+------------+
+|                     * |                           J01NME |      81015 |
+|      d45e64fa83f81cf7 |                           J01NME |     101204 |
+| 0x0000000000000abc/12 |                           J01NME |     114360 |
```

Comment:
Why does `DISCERNER` start with `0x`, but `EUI64` does not.
Is there any reason for that？ @dismirlian @abtink

Reply:
Well, just because of consistency with these:
```
> commissioner joiner add d45e64fa83f81cf7 J01NME
...

> commissioner joiner add 0xabc/12 J01NME
...

> joiner discerner
0xabc/12
Done

> joiner id
d65e64fa83f81cf7
Done
```